CollAction/Services/Image/ImageService.cs

```diff
@@ -0,0 +1,153 @@
+﻿using CollAction.Data;
+using CollAction.Models;
+using SixLabors.ImageSharp;
+using Microsoft.AspNetCore.Http;
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using SixLabors.ImageSharp.PixelFormats;
+using Microsoft.Extensions.Options;
+using Amazon.S3;
+using Amazon.S3.Model;
+using Amazon;
+using Hangfire;
+using Microsoft.EntityFrameworkCore;
+using System.Linq;
+using CollAction.Helpers;
+using Microsoft.AspNetCore.Hosting;
+
+namespace CollAction.Services.Image
+{
+    public class ImageService : IImageService
```

Comment:
Service could have Amazon specific name ?

Reply:
It's AmazonS3ImageService now, interface name remains the same.